app/src/main/java/org/mozilla/fenix/components/TopSiteStorage.kt

```diff
@@ -86,4 +87,10 @@ class TopSiteStorage(private val context: Context) {
             context.settings().defaultTopSitesAdded = true
         }
     }
+
+    fun prefetch() {
```

Comment:
nit: I think we need to explain why we do this. We should probably only do so once in the code so maybe say to reference some other method (or the other explanation should reference here?)

We should probably also explain why this is innocuous.